src/components/TextInput/TextInput.tsx

```diff
@@ -42,6 +43,7 @@ export const TextInput = React.forwardRef<HTMLSpanElement, TextInputProps>(funct
         name,
         value,
         defaultValue,
+        innerLabel,
```

Comment:
We call this prop in `<Select/>` just `label`. The behaviour should be also identical.